### DISCUSSION TITLE
Tags have no corresponding links?

### DISCUSSION THREAD

**Discussion by diongbridger** [srid/neuron#660] [General]
Hi all,

From neuron's documentation ( https://neuron.zettel.page/tags ) I see the following: 
"In the above example, the note is tagged with #cognitive-distortion which also links to the tag page."

That '#cognitive-distortion' tag does not link to any tag page however. Nothing happens when that text is clicked on - there is no hyperlink. My personal experiments with neuron match this behaviour.

Is this 'tag page' feature currently unimplemented, or does it perhaps depend on a plugin that needs to be enabled?
My goal is to use hierarchical tags to provide a "status" to work-in-progress zettels ( e.g. draft / complete / abandoned ). It seemed like structured tags could answer this purposes ( e.g. status/draft ) and the 'status' tag itself could have some short text explaining the tag schema.

I'd be grateful for any assistance on this.

**Comment by srid** [comment]
Yea, search got removed in `master`. Once we bring it back (see https://github.com/srid/neuron/issues/567), tag links will start working again.

But since neuron is in maintenance mode, I recommend checking out its successor instead: https://note.ema.srid.ca/ - it is [more capable](https://note.ema.srid.ca/guide/neuron) than neuron in several ways. It's got a pretty good tagging system too; [see demo](https://www.srid.ca/-/tags).
